colors/build.js

```diff
@@ -2,29 +2,50 @@ const fs = require('fs');
 const colors = require('./photon.json');
 const metadata = require('./package.json');
 
-function createColor(color, element) {
-  var rv = [];
+function createColor(color, element, outputType) {
+  var output = [];
+  let prefix;
+  let lastColor;
```

Comment:
`lastColor` doesn't seem to be used, so we should drop it.  But I think we can fold that into a larger change…
Since the only thing we're changing here is the prefix, _and_ since a couple of different syntaxes use the `$`, let's just specify that in the parameters, with a default of `'  --'`, and get rid of the switch statement.
And while we're at it, let's change `output` back to `rv`, to minimize the diff. 😉